common/global_cpu_executor.cpp

```diff
@@ -23,10 +23,14 @@
 
 #include "wangle/concurrent/CPUThreadPoolExecutor.h"
 
+DEFINE_int32(global_worker_threads, sysconf(_SC_NPROCESSORS_ONLN),
+             "The number of threads for global CPU executor.");
+
```

Comment:
I meant to say getGlobalCPUExecutor().
We used to have a gflag for exactly the same purpose, and we hit this tricky bug in Scorpion. So it got removed eventually.